client-sdk/ts-web/core/src/consensus.ts

```diff
@@ -115,7 +115,7 @@ export async function hashSignedTransaction(signed: types.SignatureSigned) {
 
 export class TransactionWrapper<BODY> {
     transaction: types.ConsensusTransaction;
-    signedTransaction: types.SignatureSigned;
+    signedTransaction?: types.SignatureSigned;
```

Comment:
:skull: there's no way to do `wrapper.signedTransaction!` in javascript

Reply:
ew
I would choose wrong type then
```suggestion
    signedTransaction: types.SignatureSigned = undefined as any;
```

Reply:
turns out you can do

```js
/** @type {oasis.types.SignatureSigned} */ (tw.signedTransaction)
```

to cast away the `| undefined` ness

Reply:
ok mostly using `null as never` in this new draft